src/stylesheets/core/_utilities.scss

```diff
@@ -150,7 +152,7 @@
       z-index: 1;
     }
 
-    &.usa-current {
+    &.usa-current { /* stylelint-disable-line selector-no-qualifying-type */
```

Comment:
Actually, it might be required because things like `a:visited` might override it.